src/shortcuts/__tests__/undo.ts

```diff
@@ -341,3 +341,37 @@ it('non-undoable actions are ignored', () => {
   expect(store.getState().inversePatches.length).toEqual(0)
 
 })
+
+it('undo splitSentences', () => {
+  const store = createTestStore()
+
+  store.dispatch([
+    {
+      type: 'importText',
+      path: RANKED_ROOT,
+      text: `
+        - First Sentence. Second Sentence
+        - t`
+    },
+    { type: 'setCursor', path: [{ value: 'First Sentence. Second Sentence', rank: 0 }] },
+    { type: 'splitSentences' }
+  ])
```

Comment:
I think you may have accidentally branched off your other PR. You can remove this test as described in https://github.com/cybersemics/em/pull/907/#issuecomment-727037674. Thanks!

Reply:
That's correct. Removed it now